lib/utils/getServerlessConfigFile.js

```diff
@@ -59,7 +65,6 @@ const getServerlessConfigFile = _.memoize(srvcPath => getServerlessConfigFilePat
     }
 
     return '';
-  })
-);
+  }), (configFileName, srvcPath) => srvcPath);
```

Comment:
Is this ok that `configFileName` is ignored here? 

Due to that calling function mulitple times with same `srvcPath` but different `configFileName` arguments, will produce same results.

What case does this memoization addresses?

Reply:
right. good point. The memoization is just to avoid loading the file more than once in cases of... probably badly engineered code 😬 

i'll fix the key specification.